apps/emqx_ds_builtin_raft/src/emqx_ds_replication_layer_shard.erl

```diff
@@ -336,21 +372,50 @@ handle_cast(_Msg, State) ->
     {noreply, State}.
 
 terminate(_Reason, {DB, Shard}) ->
+    %% FIXME
```

Comment:
Stale comment?

Reply:
Sort of. Fixed in fa4d3648f994387d67e593e3141a31651f2006c9.